_episodes/02-setup.md

```diff
@@ -73,7 +73,41 @@ For this lesson, we will be interacting with [GitHub](https://github.com/) and s
 > $ git config --global core.autocrlf true
 > ~~~
 > {: .language-bash}
-> 
+>
+{: .callout}
+
+> ## Default Pull Behavior
+>
+> As of `git` version 2.27 in order to avoid a distracting hint from being
+> displayed later in this lesson, the following setting is recommended:
+>
+> ~~~
+> $ git config --global pull.rebase false
+> ~~~
+> {: .language-bash}
+>
+> Without this setting you may see this hint text,
```

Comment:
I would remove this block of text about what will happen if you don't add this config. I think it's just scary and overwhelming. I think our learners don't need more information at this point in the set up.

Reply:
My thought in adding this was that it would allow users to find this by searching for parts of the hint text.  (I'm not sure how reliably students/users follow the setup instructions, or if they are liklely to just jump into things.)  If we take the view that "all" users will follow these setup instructions, then I agree we don't need the hint text.

Reply:
I agree that we don't need the entire text. Could the lesson be rephrased to 'you may see a help text from git', and not repeat it?